### DISCUSSION TITLE
How to colorify the Github Profile?

### DISCUSSION THREAD

**Discussion by StundnerGabriel** [community/community#60236] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

I'm starting using Github and I find out that we can put colors on Github README, How this work?

**Comment by F4NT0** [comment, marked as answer]
Github support mathematical equations using [Katex](https://katex.org/docs/supported.html)
![Math-Support-Social-2x](https://github.com/community/community/assets/18719295/b60b45d2-2b21-432a-a81d-9309a4f92455)

So to insert color we use the `\color{colorname}` inside the math block.

- One example:

```latex
$$\color{orange} Hello $$
```

- In Github markdown:

$$\color{orange} Hello $$

---

- You can put color inside the text or centralize:

```latex
Hello, my name is $\color{orange} Peter$! nice to meet you. 

# $$\color{magenta} Tittle$$
```

- Working example:

Hello, my name is $\color{orange} Peter$! nice to meet you. 

# $$\color{magenta} Tittle$$

---

- You can change the font and put inside a box if you wanted:
- OBS: don't forget to use `\space` between words

```latex
## $$\sf Different \space Font$$
## $$\frak{Other \space Different \space Font}$$
```

- Live example:

## $$\sf Different \space Font$$
## $$\frak{Other \space Different \space Font}$$

- With a box:

```latex
## $$\boxed{\color{lightgreen} Hello \space World}$$
```

Example:

## $$\boxed{\color{lightgreen} Hello \space World}$$

**Reply by StundnerGabriel** [reply]
Thanks bro, it help's a lot ❤️
